source/includes/v2/_odsql.md

```diff
@@ -94,10 +109,10 @@ my_boolean_field is TRUE
 my_boolean_field: FALSE
 ```
 
-Boolean literals should be used in boolean filters and can be either TRUE or FALSE keywords (non case sensitive)
+A boolean literal can either be a TRUE or a FALSE keyword (non case sensitive). It should be used in boolean filters.
```

Comment:
non case sensitive = case insensitive?

Reply:
Yes, but this is because I am not an insensitive person